runtime/src/main/java/org/corfudb/runtime/view/LayoutManagementView.java

```diff
@@ -161,8 +162,10 @@ public void addNode(Layout currentLayout,
             // Add node is successful even if reconfigure sequencer fails.
             // TODO: Optimize this by retrying or submitting a workflow to retry.
             reconfigureSequencerServers(currentLayout, newLayout, true);
-        } catch (InterruptedException | ExecutionException e) {
-            log.error("Bootstrapping sequencer failed due to exception : ", e);
+        } catch (InterruptedException ie) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either re-interrupt this method or rethrow the "InterruptedException". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2142)